### PR TITLE
Port 1.7.2-dcos to 1.9.4

### PR DESCRIPTION
**Pull request by branden**
This rebases the customizations on 1.7.2-dcos onto the tagged 1.9.4 release.

https://jira.mesosphere.com/browse/DCOS_OSS-4675

DC/OS PR: https://github.com/dcos/dcos/pull/4439

### CODE REVIEW

**Review by philipnrmn — DISMISSED** [review]

File: plugins/inputs/mesos/mesos.go [hunk 1]
```diff
@@ -34,34 +36,109 @@ type Mesos struct {
...
+	EventType     string
+	OperationType string
+	TaskState     string
+	RoleName      string
+	FieldName     string
+	Resource      string
+	Value         interface{}
+}
+
+func (tf TaggedField) hash() string {
```
Comment (gracedo) [inline plugins/inputs/mesos/mesos.go:61]: Can you pull in the recent changes in dcos-telegraf? (bug fix for this hashing function)

**Review by gracedo — CHANGES_REQUESTED** [review]
LGTM! Just one comment to pull in the latest changes to 1.7.2-dcos

File: plugins/inputs/mesos/mesos.go (hunk 1 above)
Comment (branden) [reply plugins/inputs/mesos/mesos.go:61]: @gracedo I added #40, PTAL.

**Review by gracedo — APPROVED** [review]
💃

**Review by philipnrmn — APPROVED** [review]

**Pull request merged**